src/containers/Headers.js

```diff
@@ -27,7 +28,11 @@ const handlers = {
   updateSort: props => event => {
     const isAsc = !hasClass('sorted-asc', event.target.className)
     const id = event.target.getAttribute('data-id')
-    props.refreshData(props.name, {...props.filter, sorts: updateFilterSort(id, isAsc, props.filter.sorts)})
+    const updatedFilter = {...props.filter, sorts: updateFilterSort(id, isAsc, props.filter.sorts)}
+    if (updatedFilter && !updatedFilter.page) {
+      props.refreshCount(props.name, updatedFilter)
+    }
+    props.refreshData(props.name, updatedFilter)
```

Comment:
We aren't careful this will result in two XHR calls.

Reply:
not sure what you mean